### DISCUSSION TITLE
classes/class_navigationregion2d

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#288] [User-contributed notes]
# classes/class_navigationregion2d

Experimental: This class may be changed or removed in future versions. Inherits: Node2D< CanvasItem< Node< Object A traversable 2D region that NavigationAgent2D s can use for pathfinding. Descripti...

https://docs.godotengine.org/en/stable/classes/class_navigationregion2d.html

**Comment by savipauk** [comment]
Useful note here is that if you add StaticBody2D nodes (with CollisionShape2D children, of course) as children to this node, they will automatically be cut out of the NavigationPolygon when baking it. This is implied in the docs for NavigationPolygon, but very easy to miss.

**Reply by markopolojorgensen** [reply]
Same is true for Polygon2D nodes.
